src/handlers/query-user.ts

```diff
@@ -0,0 +1,62 @@
+import { Context } from "../types/context";
+
+export async function queryUser(context: Context, username: string) {
+  const {
+    octokit,
+    payload,
+    adapters: { supabase },
+    config,
+  } = context;
+  const body: string[] = [];
+  try {
+    const {
+      data: { id },
+    } = await octokit.users.getByUsername({
+      username,
+    });
+    if (!config.allowPublicQuery && payload.organization) {
+      const { status } = await octokit.orgs.checkMembershipForUser({
+        username,
```

Comment:
You need to check the membership of the user who posted the comment not the user who is queried or maybe I'm missing something